tasks_management/services.py

```diff
@@ -228,6 +234,9 @@ def _adjust_update_task_data(self, obj_data):
                 obj_data[key] = str(obj_data[key])
         return obj_data
 
+    def _data_for_json_ext_update(self, obj_data):
```

Comment:
Same as on line 188. Moreover that method could be unified to have one signature both for create/update context. Seems that method behaves in the same way for both contexts.